ext/standaloneusers/Civi/Api4/Action/User/WriteTrait.php

```diff
@@ -0,0 +1,107 @@
+<?php
+namespace Civi\Api4\Action\User;
+
+use Civi\API\Exception\UnauthorizedException;
+
+trait WriteTrait {
+
+  /**
+   * If a password is given in the values to write, should we hash it?
+   *
+   * - 'system' when the password you are providing is in plaintext and
+   *    therefore needs to be hashed before it is written to the database
+   * - 'raw' when you want to write the value in password to the database
+   *    as-is. e.g. if you have somehow done your own hashing, or more likely
+   *    you loaded the record from the database and so already have a hashed
+   *    version.
+   * - 'guess' (default) **Do not use this!** It is there to prevent
+   *    hashing-what-was-already-hashed accidents, but you should be explicit
+   *    about what you want the API to do with the password you are providing.
```

Comment:
Now that my beard is long and grey I can speak from the wisdom of the ages to say that I have *never* regretted making new APIs very strict (strictness can always be loosened later) but I have often (just yesterday in fact) regretted having made them too lax/forgiving because years later you'll be stuck with that laxness and there's nothing you can do about it.

So, I'm not exactly sure what this does, but since this is a brand-new api and this appears to be a loosey-goosey setting... are you sure you want to start off with this?